.env.docker

```
@@ -16,4 +16,8 @@ HAWK_ENABLED=true
 HAWK_URL=wss://collector:3000/ws
 HAWK_BULK_SIZE=5
 HAWK_SOURCE_CODE_ENABLED=false
-HAWK_TOKEN=
\ No newline at end of file
+HAWK_TOKEN=
+
+REDIS_URL=localhost:6379
+REDIS_PASSWORD=
+REDIS_SET=
```

```suggestion
REDIS_SET=DisabledProjectsSet
```
давай добавим значение по дефолту
и мне кажется стоит переменовать переменную, чтобы было лучше понятно что в ней лежит
типо REDIS_DISABLED_PROJECT_SET
ну или мб как-то по другому